1/ToDoApp/src/backend/schema/typeDefs.js

```diff
@@ -0,0 +1,32 @@
+const { gql } = require('apollo-server');
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
+// your data.
+const typeDefs = gql`
+  type Todo {
+    id: Int!
+    message: String
+    finished: Boolean
+    creator: [User]
+  }
+  type User {
+    id: Int!
+    name: String
+    email: String
+    password: String
+  }
+  type Query {
+    todoById(id: Int!): Todo
+    allTodos: [Todo]
+    allUsers: [User]
+  }
+  type Mutation {
+    addTodo( message: String, token: String): [Todo]
```

Comment:
It's much more common to return the created todo instead of the entire list of todos.

```suggestion
    addTodo( message: String, token: String): Todo
```